src/react/Reg.tsx

```diff
@@ -1,9 +1,9 @@
-import Registration from "./Registration.js";
-import Authorization from "./Authorization.js";
+import Registration from "./Registration";
+import Authorization from "./Authorization";
 import React from "react";
 
-function Reg() {
-    let [change, setChange] = React.useState(<Authorization />)
+const Reg: React.FC = () => {
```

Comment:
Consider renaming the component. So that it is clear by name what is inside the component